server/src/routes/cart.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const router = express.Router();
+const checkSession = require('../middleware/checkSession');
+const IndexCartController = require('../controllers/Cart/IndexCartController');
+const UpdateCartController = require('../controllers/Cart/UpdateCartController');
+const DeleteCartItemController = require('../controllers/Cart/DeleteCartItemController');
+const EmptyCartController = require('../controllers/Cart/EmptyCartController');
+
+module.exports = app => {
+    const redisClientService = app.get('redisClientService');
+
+    const indexCartController = new IndexCartController(redisClientService);
```

Comment:
you are in `cart.js` so in my opinion it should be called `indexController` `updateController` etc.

Reply:
done